src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -5,40 +5,57 @@
 import java.nio.file.Files;
 import java.nio.file.Paths;
 import java.util.Arrays;
+import java.util.Comparator;
 import java.util.List;
 import java.util.stream.Collectors;
 
 public class StreamStudy {
 
-    public static long countWords() throws IOException {
-        String contents = new String(Files.readAllBytes(Paths
-                .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
-        List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
+  public static long countWords() throws IOException {
+    String contents = new String(Files.readAllBytes(Paths
+        .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
+    List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        long count = 0;
-        for (String w : words) {
-            if (w.length() > 12) count++;
-        }
-        return count;
+    long count = 0;
+    for (String w : words) {
+      if (w.length() > 12) {
+        count++;
+      }
     }
-
-    public static void printLongestWordTop100() throws IOException {
-        String contents = new String(Files.readAllBytes(Paths
-                .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
-        List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
-
-        // TODO 이 부분에 구현한다.
-    }
-
-    public static List<Integer> doubleNumbers(List<Integer> numbers) {
-        return numbers.stream().map(x -> 2 * x).collect(Collectors.toList());
-    }
-
-    public static long sumAll(List<Integer> numbers) {
-        return numbers.stream().reduce(0, (x, y) -> x + y);
-    }
-
-    public static long sumOverThreeAndDouble(List<Integer> numbers) {
-        return 0;
+    return count;
+  }
+
+  public static void printLongestWordTop100() throws IOException {
+    String contents = Files.readString(Paths.get("src/main/resources/fp/war-and-peace.txt"));
+    List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
+
+    // TODO 이 부분에 구현한다.
+    List<String> newWords = words.stream()
+        .sorted(Comparator.comparingInt(String::length)
+            .reversed()
+        )
+        .distinct()
+        .limit(100)
+        .map(String::toLowerCase)
+        .collect(Collectors.toUnmodifiableList());
+
+    for (String word : newWords) {
+      System.out.println(word);
     }
-}
\ No newline at end of file
+  }
+
+  public static List<Integer> doubleNumbers(List<Integer> numbers) {
+    return numbers.stream().map(x -> 2 * x).collect(Collectors.toList());
+  }
+
+  public static long sumAll(List<Integer> numbers) {
+    return numbers.stream().reduce(0, (x, y) -> x + y);
```

Comment:
```suggestion
    return numbers.stream()
    .sum()
```

위와 같이 표현 할 수도 있으니 참고하시면 좋을 것 같아요.